.travis.yml

```diff
@@ -28,75 +28,16 @@ env:
 
 install:
     - if ([ "$MODULE" != "demo-shell" ]); then
-        if ([ "$TRAVIS_BRANCH" = "master" ]); then
```

Comment:
Why this PR removes half of the travis setup?

Reply:
I guess @bbcodrin created this branch starting from a wrong one not development